sigmf-spec.md

```diff
@@ -663,22 +663,42 @@ this field be capable of displaying up to 20 characters.
 ### Collection Format
 
 The `sigmf-collection` file contains metadata in a single top-level object 
-called a `collection`. The Collection Object contains key/value pairs that 
-describe relationships between SigMF Recordings.
-
-The Collection object points to specific Recordings via a `SigMF Recording tuple`,
-which references the base-name of the Recording and the SHA512 hash of the 
-Metadata file. Tuples MAY be the singular value in a key/value pair, or provided
-in an ordered list via a JSON array.
+called a `collection`. The Collection object associates specific Recordings
+together through key/value pairs which link metadata keys and
+`SigMF Recording Objects` that point to Recordings containing actual data. The
+following rules apply to SigMF Collections:
 
 1. The Collection Object MUST be the only top-level object in the file.
-1. The Collection Object MUST only contain SigMF key/value pairs.
-1. Keys in the Collection Object MUST use SigMF Recording Tuples to reference 
+1. The Collection Object MUST only contain SigMF key/value pairs provided by the
+   core specification or a compliant SigMF extension.
+1. Keys in the Collection Object MUST use SigMF Recording Objects to reference
    Recordings.
-1. SigMF Recording Tuples MUST take the form of `["N", "hash"]`, where `N` is the 
-   base-name of a SigMF Recording and `hash` is the SHA512 hash of the Recording's 
-   Metadata file `N.sigmf-meta`.
-1. If a value contains multiple SigMF Recording Tuples, they MUST appear in a JSON array.
+1. SigMF Recording Objects MUST contain both a `name` which is the base-name of
+   a SigMF Recording and a `hash` which is the SHA512 hash of the Recording
+   Metadata file `[base-name].sigmf-meta`.
+1. If a value contains multiple SigMF Recordings, they MUST appear in a JSON array.
+
+`SigMF Recording Objects` reference the base-name of the SigMF Recording and the
```

Comment:
This should be a sub-section with a table describing the required fields in the Object, like we do elsewhere in the spec. Reading it, I realize we should have previously done this for the tuple version, too, but neglected it at the time.